x-pack/plugin/eql/src/test/build.gradle

```diff
@@ -0,0 +1,3 @@
+dependencies {
+  compile 'io.ous:jtoml:2.0.0'
+}
```

Comment:
can remove this after applying the change above

Reply:
thank you!